### PR TITLE
MC-10527/ Create Security Rule doc

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
Documentation for creating a network security rule in a network security group

### CODE REVIEW

**Review by lamminade — APPROVED** [review]
LGTM

File: source/includes/azure/_security_rules.md [hunk 1]
```diff
@@ -109,19 +112,75 @@ Attributes | &nbsp;
...
+  "securityGroupId": "/subscriptions/subscription/resourceGroups/example-system-azure-example/providers/Microsoft.Network/networksecuritygroups/sample-network-security-group",
+  "name": "SampleRule",
+  "priority": 110,
+  "direction": "Inbound",
+  "access": "Deny",
+  "protocol": "Udp",
+  "sourcePortRanges": [
+    "1024"
+  ],
+  "sourceAddressPrefixes": [
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md]: Shouldn't it be sources

File: source/includes/azure/_security_rules.md [hunk 2]
```diff
@@ -109,19 +112,75 @@ Attributes | &nbsp;
...
+    "0.0.0.1",
+    "10.0.0.0/32"
+  ],
+  "destinationPortRanges": [
+    "8080"
+  ],
+  "destinationAddressPrefixes": [
```
Comment (fyrnaga) [inline source/includes/azure/_security_rules.md]: shouldn't it be destinations

File: source/includes/azure/_security_rules.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_security_rules.md]: yes, forgot to changed

File: source/includes/azure/_security_rules.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_security_rules.md]: yes!

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**